app/backend/src/couchers/servicers/pages.py

```diff
@@ -52,9 +53,9 @@ def _can_moderate_page(session: Session, page: Page, user_id: int) -> bool:
     return can_moderate_node(session, user_id, page.parent_node_id)
 
 
-def page_to_pb(session: Session, page: Page, context) -> pages_pb2.Page:
-    first_version = page.versions[0]
-    current_version = page.versions[-1]
+def page_to_pb(session: Session, page: Page, context: CouchersContext) -> pages_pb2.Page:
+    first_version = page.versions[0]  # type: ignore[attr-defined]
```

Comment:
`backref` breaks type annotations, I plan to convert to `back_populates` in a future PR